Firestore/core/src/firebase/firestore/auth/token.cc

```diff
@@ -21,15 +21,15 @@ namespace firestore {
 namespace auth {
 
 Token::Token(const absl::string_view token, const User& user)
-    : token_(token), user_(user), is_valid_(true) {
+    : token_(token), user_(user) {
 }
 
-Token::Token() : token_(), user_(User::Unauthenticated()), is_valid_(false) {
+Token::Token() : token_(), user_(User::Unauthenticated()) {
```

Comment:
Is it reasonable to just wholesale avoid having this constructor?

We want callers to use `Token::Unauthenticated` when they mean this.